lend/src/components/WalletPage/index.tsx

```diff
@@ -247,8 +333,16 @@ interface WalletPageProps {
 
 export const WalletPage = ({ onHide }: WalletPageProps) => {
   const { active, account, activate, deactivate } = useWeb3Wallet();
+  let initialWalletPageStep = undefined;
```

Comment:
Addressed in [da914ae](https://github.com/ribbon-finance/ribbon-frontend/pull/624/commits/da914ae3a10539aa9d119085508d60ebfdf41c86), but I'm not sure if this is the right way. Reverted back to the original design and logically I think it makes sense, but I didn't useEffect/useMemo since I think the state gets set every time the component renders?